scripts/zones/Batallia_Downs_[S]/mobs/Chaneque.lua

```diff
@@ -0,0 +1,45 @@
+-----------------------------------
+-- Area: Batallia Downs (S)
+--  NM:  Chaneque
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_ADD_EFFECT,mob:getShortID());
+end;
+
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+function onAdditionalEffect(mob,target,damage)
+    local chance = 10;
+    
+    if (math.random(0,99) >= chance) then
+        return 0,0,0;
+    else
+        local power = 10;
+        local params = {};
+        params.bonusmab = 0;
+        params.includemab = false;
+        power = addBonusesAbility(mob, ELE_DARK, target, power, params);
+        power = power * applyResistanceAddEffect(mob,target,ELE_DARK,0);
+        power = adjustForTarget(target,power,ELE_DARK);
+        if (power < 0) then
+            power = 0
+        end
+        power = finalMagicNonSpellAdjustments(mob,target,ELE_DARK,power);
```

Comment:
oh actually..this needs to be before that check for < zero. sorry I missed it before.